docs/cookbooks/common_workflows.md

```diff
@@ -284,3 +284,20 @@ To generate and check the docs locally:
 1. Run `./bqetl docs generate --output_dir generated_docs`
 1. Navigate to the `generated_docs` directory
 1. Run `mkdocs serve` to start a local `mkdocs` server.
+
+
+## Setting up change control to code files
+
+Each code files in the bigquery-etl repository can have a set of owners who are responsible to review and approve changes, and are automatically assigned as PR reviewers.
+The query files in the repo also benefit from the metadata labels to be able to validate and identify the data that is change controlled.
+
+1. Select or create a [Github team or identity](https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team) and add the GitHub emails of the query codeowners. A GitHub identity is particularly useful when you need to include non @mozilla emails or to randomly assign PR reviewers from the team members. This team requires edit permissions to bigquery-etl, to achieve this, inherit the team from one that has the required permissions e.g. `mozilla > telemetry`.
+1. Open the `metadata.yaml` for the query where you want to apply change control:
```

Comment:
An example PR or code block might be useful here, so folks can copy-paste

Reply:
Agree, added a sample PR. Will do a Data Club talk as well.